spec/index.bs

```diff
@@ -1067,28 +1064,21 @@ Note: The HTML Standard defines how the validation is performed. [[!HTML]]
 
 ## Integration with HTML ## {#integration-with-html}
 
-{{Window}} objects have a <dfn for="Window">trusted type policy factory</dfn>,
+{{Window}} and {{Worker}} objects have a <dfn for="Window">trusted type policy factory</dfn>,
 which is a {{TrustedTypePolicyFactory}} object.
 
-Issue: Define for workers.
-
-### Extensions to the Window interface ### {#extensions-to-the-window-interface}
+### Extensions to the WindowOrWorkerGlobalScope interface ### {#extensions-to-the-windoworworkerglobalscope-interface}
 
-This document extends the {{Window}} interface defined by [[HTML5|HTML]]:
+This document extends the {{WindowOrWorkerGlobalScope}} interface defined by [[HTML5|HTML]]:
 
 <pre class="idl">
-partial interface Window {
+partial interface WindowOrWorkerGlobalScope {
```

Comment:
I've sent https://github.com/w3c/webappsec-trusted-types/pull/334 to fix a problem introduced here.